tests/io_utils.go

```diff
@@ -45,32 +46,72 @@ func NodeNameWithHandler() string {
 	return node.ObjectMeta.Name
 }
 
-func ExecuteCommandInVirtHandlerPod(nodeName string, args []string) error {
+func ExecuteCommandInVirtHandlerPod(nodeName string, args []string) (string, error) {
 	virtClient, err := kubecli.GetKubevirtClient()
 	if err != nil {
-		return err
+		return "", err
 	}
 
 	pod, err := kubecli.NewVirtHandlerClient(virtClient).Namespace(flags.KubeVirtInstallNamespace).ForNode(nodeName).Pod()
 	if err != nil {
-		return err
+		return "", err
 	}
 
 	stdout, stderr, err := ExecuteCommandOnPodV2(virtClient, pod, "virt-handler", args)
 	if err != nil {
-		return fmt.Errorf("Failed excuting command=%v, error=%v, stdout=%s, stderr=%s", args, err, stdout, stderr)
+		return "", fmt.Errorf("Failed excuting command=%v, error=%v, stdout=%s, stderr=%s", args, err, stdout, stderr)
+	}
+	return stdout, nil
+}
+
+func CreateErrorDisk(nodeName string) (string, string) {
+	// Todo this will only work on fresh nodes
+	args := []string{"/usr/bin/virt-chroot", "--mount", "/proc/1/ns/mnt", "exec", "--", "/usr/sbin/modprobe", "scsi_debug", "opts=2", "every_nth=4"}
```

Comment:
is the `scsi_debug` module installed in the envrironment?

Reply:
Yes.